weave-js/src/components/Button/Button.tsx

```diff
@@ -6,6 +6,7 @@
 
 import classNames from 'classnames';
 import React, {ReactElement} from 'react';
+import {PopupProps} from 'semantic-ui-react';
```

Comment:
Sad that we can't get rid of semantic UI.

Reply:
if I had more time, i would've replaced the semantic popup with our radix tooltip 🙈

Reply:
You know what, you've inspired me. I might as well do it now while I'm touching this code. Please hold

Reply:
Didn't end up taking long at all :tada: updated to use radix tooltip instead of semantic one